### PR TITLE
Update to Disruptions to Service Availability

### PR DESCRIPTION
**Pull request by RichardEustace**

### CODE REVIEW

File: docs/_articles/disruptions-to-service-availability.md [hunk 1]
```diff
@@ -10,6 +10,19 @@ This page lists all known occurrences of Street Manager service being unavailabl
 
 To see the current status of Production and Sandbox environments visit the [status page here ](http://stats.pingdom.com/uyxey83hvki0)
 
+## May 2020
+
+**Production environment**
+
+* no disruptions to availability
```
Comment (cpaul92) [inline docs/_articles/disruptions-to-service-availability.md:17]:
```suggestion
* No disruptions to availability
```

File: docs/_articles/disruptions-to-service-availability.md [hunk 2]
```diff
@@ -10,6 +10,19 @@ This page lists all known occurrences of Street Manager service being unavailabl
...
 
+## May 2020
+
+**Production environment**
+
+* No disruptions to availability
+
+**Sandbox environment**
+
+* 16/05/2020 04:50 to 07:57: Sandbox environment unavailable for 3 Hours 7 mins due to unexpected outage.
```
Comment (matthewdoreilly) [inline docs/_articles/disruptions-to-service-availability.md:21]:
```suggestion
* 16/05/2020 04:50 to 07:57: Sandbox environment unavailable for 3 hours 7 mins due to unexpected outage.
```

File: docs/_articles/disruptions-to-service-availability.md [hunk 3]
```diff
@@ -10,6 +10,19 @@ This page lists all known occurrences of Street Manager service being unavailabl
...
+
+* 16/05/2020 08:16 to 08:19: Sandbox environment unavailable for 3 mins due to unexpected outage.
+
+* 17/05/2020 20:45 to 18/05/2020 08:37: Sandbox environment unavailable for 11 Hours 53 mins due to unexpected outage.
```
Comment (matthewdoreilly) [inline docs/_articles/disruptions-to-service-availability.md:25]:
```suggestion
* 17/05/2020 20:45 to 18/05/2020 08:37: Sandbox environment unavailable for 11 hours 53 mins due to unexpected outage.
```

File: docs/_articles/disruptions-to-service-availability.md [hunk 4]
```diff
@@ -10,6 +10,19 @@ This page lists all known occurrences of Street Manager service being unavailabl
...
 To see the current status of Production and Sandbox environments visit the [status page here ](http://stats.pingdom.com/uyxey83hvki0)
 
+## May 2020
+
+**Production environment**
+
+* No disruptions to availability
+
+**Sandbox environment**
+
```
Comment (matthewdoreilly) [inline docs/_articles/disruptions-to-service-availability.md:20]: I think these timings should be in descending order

**Pull request merged**